torch/csrc/Module.cpp

```diff
@@ -54,6 +54,7 @@
 #ifdef USE_DISTRIBUTED
 #ifdef USE_C10D
 #include <torch/csrc/distributed/c10d/c10d.h>
+#include <torch/csrc/distributed/rpc/rpc.h>
```

Comment:
@pietern is it OK to add this here, or is it better to have a separate `USE_RPC` flag?

Reply:
If RPC has no added dependencies over distributed, let's please roll it into `USE_DISTRIBUTED` and not add ANOTHER build option for us.